src/utils/types.cljs

```diff
@@ -0,0 +1,31 @@
+(ns utils.types
```

Comment:
`utils.types` does not seem so accurate to me.
The functionality in here is mostly some form of converter. 
perhaps something along those lines is clearer  `utils.change-type` or `utils.converters`

Reply:
rename to `utils.transforms`, based on `cljs.core/clj->js` doc string https://github.com/clojure/clojurescript/blob/c8ab636594f3914a2430b89a7d09fa1e6693e2db/src/main/cljs/cljs/core.cljs#L11032